File: Python/4. Sets/exercise12.py
# Check Strict Superset "https://www.hackerrank.com/challenges/py-check-strict-superset/problem"

# Enter your code here. Read input from STDIN. Print output to STDOUT

arr = set(map(int, input().split()))

issup = []
for i in range(int(input())):
    sup = set(map(int, input().split()))
    issup.append(arr.issuperset(sup))

print(all(issup))

File: Java/1. Introduction/Exercise12.java
// Java End-of-file "https://www.hackerrank.com/challenges/java-end-of-file/problem"

import java.util.Scanner;


public class Exercise12 {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner sc = new Scanner(System.in);
        int i = 1;
        while (sc.hasNext()) {
            System.out.println(i + " " + sc.nextLine());
            i++;
        }
    }
}
